pages/common/vegeta.md

```diff
@@ -15,14 +15,14 @@
 
 `echo "{{GET https://example.com}}" | vegeta attack -duration={{30s}} -rate={{10}}`
 
-- Launch an attack and display for a report:
+- Launch an attack and display a report:
```

Comment:
This is a remnant of a change from "ask for" to "display", suggested by @sbrl [here](https://github.com/tldr-pages/tldr/pull/2519#discussion_r228876529). Looks like the change left the "for" behind.